website/content/docs/secrets/kmip.mdx

```diff
@@ -66,6 +91,43 @@ allowed operations for it.
       Success! Data written to: kmip/scope/my-service/role/admin
     ```
 
+#### Supported KMIP Operations
```

Comment:
nit: Should this really be underneath Scopes And Roles? I would have thought it better under the Usage header directly? I.E. 

```suggestion
### Supported KMIP Operations
```

Reply:
Yes, I think that is reasonable.